api/ipsec/read

```diff
@@ -66,10 +66,11 @@ if ($cmd eq 'tunnels') {
     my @interfaces;
     my $ndb = esmith::NetworksDB->open_ro();
     foreach ($ndb->red()) {
-        next if ($_->prop('type') eq 'xdsl' || $_->prop('type') eq 'xdsl-disabled');
+        next if ($_->prop('type') eq 'xdsl-disabled');
         my $bootproto = $_->prop('bootproto') || 'none';
+        my $ipaddr = $_->prop('ipaddr') || 'pppoe';
         if ($bootproto eq 'none') {
-            push(@interfaces, {"name" => $_->key, "address" => $_->prop('ipaddr')});
+            push(@interfaces, {"name" => $_->key, "address" => $ipaddr});
```

Comment:
A `xdsl` interface doesn't have the `bootproto` property, so the condition will not be met.
Please revert changes to line 71 and 72.

Reply:
Sorry @gsanchietti, but I observed that `$bootproto` variable is set on this line `my $bootproto = $_->prop('bootproto') || 'none';`.
If it's not set it will be set to `none`. Am I wrong?

Reply:
No, you're right! My fault, I didn't check the whole context.

The patch is good, thank you!